sigh/lib/sigh/resign.rb

```diff
@@ -27,7 +27,10 @@ def resign(ipa, signing_identity, provisioning_profiles, entitlements, version,
       # validate that we have valid values for all these params, we don't need to check signing_identity because `find_signing_identity` will only ever return a valid value
       validate_params(resign_path, ipa, provisioning_profiles)
       entitlements = "-e #{entitlements.shellescape}" if entitlements
-      provisioning_options = provisioning_profiles.map { |fst, snd| "-p #{[fst, snd].compact.map(&:shellescape).join('=')}" }.join(' ')
+      provisioning_options = provisioning_profiles.map do |fst, snd|
```

Comment:
@milch - sorry my reply was to fast.
after looking at `resign.sh` - wich actually gets called, there is a documentation about the whole `-p` paramater.

it can be passed either:

  * `-p file.mobileprovision`
  *  `-p com.nested-app=nested-app.mobileprovision`

so i think the `snd` param is only set, once you resign your extensions, like today-extension.


sounds correct?